components/pedigree/resources/src/main/resources/pedigree/model/export.js

```diff
@@ -401,22 +405,26 @@ define([
 
      if (alertUnknownGenderFound || warnAboutMissingDOB) {
          var warningText = "Pedigree can be exported, but there are warnings:\n\n\n";
-
-         var numberWarnings = false;
-         if (alertUnknownGenderFound && warnAboutMissingDOB) {
-             numberWarnings = true;
-         }
+         var warnings = [];
          if (alertUnknownGenderFound) {
-             warningText += (numberWarnings ? "1) " : "") +
-                             "BOADICEA format does not support unknown or other genders.\n\n" +
-                            "All persons of unknown or other gender were either assigned a gender "+
-                            "opposite to their partner's gender or saved as male in the export file";
+             warnings.push("BOADICEA format does not support unknown or other genders.\n\n" +
+                           "All persons of unknown or other gender were either assigned a gender " +
+                           "opposite to their partner's gender or saved as male in the export file");
          }
          if (warnAboutMissingDOB) {
-             warningText += (numberWarnings ? "\n\n\n2) " : "") +
-                            "BOADICEA requires that all individuals with cancer have their age specified or estimated.\n\n" +
-                            "A person with cancer is missing age data, data will be exported but may not be accepted by BOADICEA";
+              warnings.push("BOADICEA requires that all individuals with cancer have their year of " +
+                            "birth and age at cancer diagnosis specified or estimated\n\n" +
+                            "A person with cancer is missing age data, data will be exported but may not be accepted by BOADICEA");
+         }
+         if(warnMissingDOBUnaff) {
+             warnings.push("BOADICEA recommends that all unaffected individuals have their year of birth and" +
+                           " year of death, if applicable, specified or estimated." +
+                           " Not doing so may lead to an overestimation of risk.");
+         }
+         if(warnings.length > 1) {
+             warnings = warnings.map(function(v, i, a) { return (i + 1) + ") " + v; });
```

Comment:
This is a nice pattern, but keep in mind that [`Array.prototype.map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) is only available in IE9+. I think @allasm was using a helper for `Array.prototype.forEach()`, so that might be a consideration here as well.